pkgs/servers/matrix-synapse/matrix-appservice-irc/default.nix

```diff
@@ -23,9 +23,15 @@ ourNodePackages.package.override {
 
   dontAutoPatchelf = true;
 
+  postRebuild = ''
+    npm run build
+  '';
+
   postInstall = ''
     makeWrapper '${nodejs}/bin/node' "$out/bin/matrix-appservice-irc" \
       --add-flags "$out/lib/node_modules/matrix-appservice-irc/app.js"
+
+      cp -rv ${matrix-sdk-crypto-nodejs}/lib/node_modules/@matrix-org/matrix-sdk-crypto-nodejs $out/lib/node_modules/matrix-appservice-irc/node_modules/@matrix-org/
```

Comment:
a symlink wouldn't have worked here?

Reply:
I suspect it would have, but not sure if it really matters all too much unless you're building the package yourself.